package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "test": "NETWORK=celo pnpm exec hardhat test",
     "test:alfajores": "NETWORK=alfajores pnpm exec hardhat test",
-    "test:baklava": "NETWORK=baklava pnpm exec hardhat test",
+    "test:baklava": "NETWORK=baklava pnpm exec hardha test",
+    "build": "tsc",
```

Comment:
We should change this to `pnpm tsc` so we use the projects Typescription version instead of the global one.